wGetStarted/Excel-Best-and-Worst-Practices.md

```diff
@@ -71,6 +71,22 @@ Here you will use the scroll bar in the function wizard to alter the UseEntireRo
 
 Sometimes developers accidentally use the last cell of the 1,048,576 rows that Excel provides or write a value in the last of the 16,384 columns that is also available. When this happens, the range in use expands to a large range of cells that uses more memory and expands the work area, which can hinder performance. To improve performance, delete all rows and columns beyond the reporting area.
 
+### Images
+
+Images can quickly increase the size of an Excel report. If images must be included in a report, you can compress them to 96ppi by selecting the image, clicking the "Picture Format" tab on the Ribbon, and selecting "Compress Pictures." From here, you can select the compression rating and whether to apply the compression to the selected image or all images.
+
+Alternatively, you can set the default compression rating for all newly inserted images as explained on the [Microsoft Support Site](https://support.microsoft.com/en-us/office/change-the-default-resolution-for-inserting-pictures-in-office-f4aca5b4-6332-48c6-9488-bf5e0094a7d2#ID0EBF=Windows){:target="_blank"}{:rel="noopener noreferrer"}.
+
+### File Type
+
+The old Microsoft Excel type .xls does not compress the data in the report using saving so this type of file will be much larger. Always use the xlsl or xlsm file type for your reports.
+
+The Microsoft binary file (xlsb) type will further reduce the file size of your report. However, there are some limitations: 1) The file may be a security risk as you will not know if it contains macros. 2) You cannot make changes to the Excel Ribbon while a binary file is open. 3) The binary file type may be inoperable by third-party tools.
```

Comment:
there's something funky about the grammar in number 3. "Inoperable by..." just doesn't read right. I would suggest "inoperable in..." or "inoperable using..."

But also, what do we mean by third party tools here? It seems like that suggests someone could be opening an excel file in something other than excel, which is kind of confusing. I think maybe adding some context to what third party tools means might clear this up.

Reply:
Ok I changed the wording to "Other spreadsheet programs other than Excel may not be able to open the binary file."